common/kernel/command.cc

```diff
@@ -565,7 +565,8 @@ void CommandHandler::setupContext(Context *ctx)
 
     ctx->settings[ctx->id("arch.name")] = std::string(ctx->archId().c_str(ctx));
     ctx->settings[ctx->id("arch.type")] = std::string(ctx->archArgsToId(ctx->archArgs()).c_str(ctx));
-    ctx->settings[ctx->id("seed")] = ctx->rngstate;
+    Property seed_property(ctx->rngstate, 64);
+    ctx->settings[ctx->id("seed")] = seed_property;
```

Comment:
what about `ctx->settings[ctx->id("seed")] = Property(ctx->rngstate, 64);`? that's closer to the style we tend to use.